src/Firehose.Web/Infrastructure/CombinedFeedSource.cs

```diff
@@ -61,7 +61,16 @@ private static ISyndicationFeedSource TryLoadFeed(IAmACommunityMember tamarin, U
                     .FromUrl(uri)
                     .WithFilter(filter)
                     .Grr();
-                var dummy = feedSource.Feed; // poke it to make sure it doesn't go bang.
+
+                // Poke it, to see if we can really reach it
+                var request = (HttpWebRequest)WebRequest.Create(uri);
+                request.Method = "HEAD";
+
+                var response = (HttpWebResponse)request.GetResponse();
+
+                // If return code is not success or redirect, it's no good
+                if ((int)response.StatusCode < 200 || (int)response.StatusCode > 399)
+                    throw new Exception("Feed says no");
```

Comment:
It should be handled by the try/catch that is around it which was already in place when I got here. This logs some message to Elmah and returns null causing the feed to be excluded.